core/platform/email/gae_email_services_test.py

```diff
@@ -38,8 +44,144 @@ def test_sending_email(self):
 
         with self.swap(feconf, 'CAN_SEND_EMAILS', True):
             gae_email_services.send_mail(
-                feconf.SYSTEM_EMAIL_ADDRESS, feconf.ADMIN_EMAIL_ADDRESS,
+                feconf.SYSTEM_EMAIL_ADDRESS,
+                feconf.ADMIN_EMAIL_ADDRESS,
                 'subject', 'body', 'html', bcc_admin=False)
         messages = self.mail_stub.get_sent_messages(
             to=feconf.ADMIN_EMAIL_ADDRESS)
         self.assertEqual(1, len(messages))
+
+    def test_that_email_bcc_is_sent_when_bcc_admin_is_true(self):
```

Comment:
Do we have a test to ensure the reverse? (i.e, admin isn't bcc'd if bcc_admin=False)

Reply:
Yes, I missed that.
Thanks!